SalesforceCommerceCloud/pwa-kit
# Fix checkout header logo navigation

# Description

The CheckoutHeader is using the wrong navigation for logo navigation. It has not taken into account the link for multi-site. 



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

- `npm run start` at the retail react app
- Go to `localhost:3000/us/en-US/checkout`
- Click on the logo icon, it should navigate the users back to `localhost:3000/us/en-US` 

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
